test/integration/test-amp-analytics.js

```diff
@@ -17,7 +17,8 @@
 import {BrowserController, RequestBank} from '../../testing/test-helper';
 import {parseQueryString} from '../../src/url';
 
-// TODO(wg-analytics): These tests cause SauceLabs disconnections.
+// TODO(wg-analytics): These tests time out on Firefox and Safari
```

Comment:
Is there a tracking issue we could link to here?

Reply:
Edit: Since we have a similar comment below for a different WG, would probably be useful to have a single issue we could reference. Will make it easier to clean up the TODOs when it's fixed.